CHANGELOG.md

```diff
@@ -55,13 +56,13 @@
 - Refactors FMS [#139](https://github.com/openscope/openscope/issues/139)
     - This represents a ground-up, from scratch, re-build of the flight management system with new classes: `Fms`, `LegModel`, and `WaypointModel`
     - Introduces the `ModeController` that completely separates Altitude, Heading and Speed settings from the FMS and allowing the FMS to be in charge of the flight plan and any fixRestrictions defined for a given route
-    - Adds `Pilot` class that acts as a coordinator layer between the `AircraftCommander`, `AircraftInstanceModel`, `ModeController`, and `Fms`
+    - Adds `Pilot` class that acts as a coordinator layer between the `AircraftCommander`, `AircraftModel`, `ModeController`, and `Fms`
```

Comment:
Looks like your find and replace also included the changelog.  these should _not_ be affected by the change to `AircraftModel`